packages/admin/src/pages/scheme/[schemeId]/advert/[advertId]/[sectionId]/[pageId].page.tsx

```diff
@@ -116,6 +132,7 @@ const Page = ({
             switch (question.responseType) {
               case 'DATE': {
                 let dateValues;
+                let timeValues;
 
                 if (previousValues) {
```

Comment:
Likely need some code that populates timeValues if previousValues exists. This is so that the values are retained upon a validation error